default-recommendations/analyzers/identifier-usage-test.rkt

```diff
@@ -64,3 +64,292 @@ analysis-test: "thrice-used let-bound variable"
 @inspect - a
 @property usage-count
 @assert 3
+
+
+analysis-test: "unused module-level function definition"
+--------------------
+(define (f x) x)
+--------------------
+@inspect - f
+@property usage-count
+@assert 0
+
+
+analysis-test: "once-used module-level function definition"
+--------------------
+(define (f x) x)
+(f 1)
+--------------------
+@within - (define (f x) x)
```

Comment:
Narrow the focus of the code passed to `@within` to just the one form containing the `f` you want to use. Here, that's `(f x)`. Do the same in the other test cases.

```suggestion
@within - (f x)
```